Bottom: bca4349a52b80ac441c25e39b77869e50cc605c8
Top:    7ac34451aa66d7ff01628bfc022c3fc44e5de034
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 14:54:56 -0500

Refresh of buildbot-add-buildbot-master

---
diff --git a/roles/docker-buildbot/README.md b/roles/buildbot-master/README.md
similarity index 100%
rename from roles/docker-buildbot/README.md
rename to roles/buildbot-master/README.md
diff --git a/roles/docker-buildbot/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
similarity index 100%
rename from roles/docker-buildbot/defaults/main.yml
rename to roles/buildbot-master/defaults/main.yml
diff --git a/roles/docker-buildbot/meta/main.yml b/roles/buildbot-master/meta/main.yml
similarity index 100%
rename from roles/docker-buildbot/meta/main.yml
rename to roles/buildbot-master/meta/main.yml
diff --git a/roles/docker-buildbot/tasks/main.yml b/roles/buildbot-master/tasks/main.yml
similarity index 100%
rename from roles/docker-buildbot/tasks/main.yml
rename to roles/buildbot-master/tasks/main.yml
diff --git a/roles/docker-buildbot/vars/main.yml b/roles/buildbot-master/vars/main.yml
similarity index 100%
rename from roles/docker-buildbot/vars/main.yml
rename to roles/buildbot-master/vars/main.yml
